src/app/pages/psbt-request/components/page-top.tsx

```diff
@@ -0,0 +1,31 @@
+import { memo } from 'react';
+
+import { getPsbtPayloadFromToken } from '@app/common/psbt/requests';
+import { BtcIcon } from '@app/components/icons/btc-icon';
+import { Flag } from '@app/components/layout/flag';
+import { Caption, Title } from '@app/components/typography';
+import { usePsbtRequestSearchParams } from '@app/store/psbts/requests.hooks';
+
+function PageTopBase() {
```

Comment:
I understand copied, but let's name something better

Reply:
It seems like we need to make these request components more shared components at some point too?